jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -0,0 +1,121 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Furnishings job processing rules after stage runs of involuntary dissolution."""
+from datetime import datetime
+from pathlib import Path
+from typing import Final
+
+from flask import Flask, current_app
+from jinja2 import Template
+from legal_api.models import Furnishing, FurnishingGroup, XmlPayload
+from legal_api.utils.legislation_datetime import LegislationDatetime
+
+
+XML_DATE_FORMAT: Final = '%B %-d, %Y'
+
+
+class PostProcessor:
+    """Processor after stage run of furnishings job."""
+
+    def __init__(self, app, furnishings_dict):
+        """Create post process helper instance."""
+        self._app = app
+        self._furnishings_dict = furnishings_dict
+        self._xml_data = {}
+        self._processed_date = LegislationDatetime.now()
+
+    def _set_meta_info(self):
+        """Set meta information for XML file."""
+        # we leave date and volume in XML blank
+        self._xml_data['effective_date'] = self._processed_date.strftime(XML_DATE_FORMAT)
+
+    def _format_furnishings(self):
+        """Format furnishing details presented in XML file."""
+        self._xml_data['furnishings'] = {}
+        for name, furnishings in self._furnishings_dict.items():
+            self._xml_data['furnishings'][name] = XmlMeta.get_info_by_name(name)
+            self._xml_data['furnishings'][name]['items'] = sorted(furnishings, key=lambda f: f.business_name)
```

Comment:
I think this sorts by business name every time an entry is added?  

If so, can we sort at the end for all `items` under a given  `self._xml_data['furnishings'][name]` entry?

Reply:
nevermind, the code works :)